react/features/video-menu/components/web/RemoteVideoMenuTriggerButton.js

```diff
@@ -218,18 +220,19 @@ class RemoteVideoMenuTriggerButton extends Component<Props> {
                 content = { content }
                 onPopoverClose = { this._onPopoverClose }
                 onPopoverOpen = { this._onPopoverOpen }
-                overflowDrawer = { this.props._overflowDrawer }
+                overflowDrawer = { _overflowDrawer }
                 position = { this.props._menuPosition }
                 ref = { this.popoverRef }>
-                {!isMobileBrowser() && (
+                {!_overflowDrawer && (
                     <span className = 'popover-trigger remote-video-menu-trigger'>
```

Comment:
Do we need this `span` in case `!_overflowDrawer` and `isMobileBrowser()` since the icon won't be rendered? If yes - why?

Reply:
we need it so that context menu for participant on ipads renders correctly to the left of it. Otherwise context menu renders on the top left of the container html element. this fixes the first item on the list.

Reply:
Thanks!